src/styles/blocks/form.css

```
@@ -0,0 +1,41 @@
+.form {
+  display: flex;
+  flex-direction: column;
+  gap: 35px;
+  font-size: 18px;
+  padding-top: 40px;
+}
+
+.form__input-container {
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+}
+
+.form__buttons {
+  display: flex;
+  justify-content: space-between;
+}
+
+.form__input {
+  line-height: 44px;
+  padding: 0 8px;
+  border: 1px solid black;
+}
```

тут норм. что можно изменить высоту по умолчанию, line-height про другое
```suggestion
.form__input {
  height: 44px;
  padding: 0 8px;
  border: 1px solid black;
}
```